WordPress/src/main/java/org/wordpress/android/ui/prefs/MyProfileActivity.java

```diff
@@ -140,18 +141,13 @@ private void updateLabel(WPTextView textView, String text) {
         return new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                DialogUtils.showMyProfileDialog(MyProfileActivity.this,
-                        dialogTitle,
+
+                ProfileInputDialogFragment inputDialog = ProfileInputDialogFragment.newInstance(dialogTitle,
                         textView.getText().toString(),
                         hint,
                         isMultiline,
-                        new DialogUtils.Callback() {
-                            @Override
-                            public void onSuccessfulInput(String input) {
-                                updateLabel(textView, input);
-                                updateMyProfileForLabel(textView);
-                            }
-                        });
+                        textView.getId());
+                inputDialog.show(getFragmentManager(), DIALOG_TAG);
```

Comment:
Let's remove this local variable I'd say and `show()` the dialog in a single line.